### DISCUSSION TITLE
Talos not on list

### DISCUSSION THREAD

**Discussion by Hak4Kidz** [hslatman/awesome-threat-intelligence#231] [Q&A]
Hi,

Curious why you left off the largest non-governmental research team off your list?

Thanks,
H4K

**Comment by jakewarren** [comment]
Sincerely not intending to be snarky but instead of asking here in the discussions why not submit a pull request contributing it? Mostly likely it was just an oversight, there are many resources from prominent organizations missing.

**Reply by Hak4Kidz** [reply]
Will do! Wanted to understand if there was a bias before going through the motions of contributing.
